fix: add the missing part of kong gateway vault's secrets versioning description (#6551)
Kong/docs.konghq.com: app/_src/gateway/kong-enterprise/secrets-management/reference-format.md

### Query

Query arguments are used to denote configuration options in a `key=value` format to the [Vault Prefix](/gateway/{{page.kong_version}}/kong-enterprise/secrets-management/reference-format/#vault-prefix)

### Version

```text
{vault://<vault-backend|entity>/<secret-id>[/<secret-key][/][?query][#version]}
                                                                     ^^^^^^^^
```

> Version, which is the fragment part of the Vault URL, is used to identify the version number of the secret stored in the vault backends which supports versioning.

Reviewer: ```suggestion
The version, specified as the fragment of the Vault URL, identifies the version number of the secret stored in a vault backend. Applies to any vault backend that supports versioning.
```
